docs/usage.md

```diff
@@ -6,28 +6,202 @@
+
+These two sub-workflows, for now, need the use of the variables `classT`, `synteny_genome_path`, `data_dir` and `geneset`. These variables are found inside the yaml ( this is the file that will tell TreeVal what and where everything is ). Currently, we don't use `common_name`, e.g., `bee`, `wasp`, `moth`, e.t.c. However, we hope to make use of it in the future as our gene_alignment_data "database" grows.
+
+First, you should set up a directory in our recommended structure:
 
 ```bash
-geneset = "Gae_host.Gae,CSKR_v2.CSKR"
 
-for organism in geneset.split(',')
-  path = {gene_alignment_dir}{classT}/{organism}/csv-data/{organism}-data.csv
+treeval-resources
+  │
+  ├─ gene_alignment_data/
+  │   └─ { classT }
+  │         ├─ csv_data
+  │         │       └─ { Name.Assession }-data.csv # Generated by our scripts
+  │         └─ { Name } # Here and below is generated by our scripts
+  │                 └─ { Name.Assession }
+  │                             ├─ cdna
+  │                             │   └─ { Chunked fasta files }
+  │                             ├─ rna
+  │                             │   └─ { Chunked fasta files }
+  │                             ├─ cds
+  │                             │   └─ { Chunked fasta files }
+  │                             └─ pep
+  │                                 └─ { Chunked fasta files }
+  │
+  ├─ gene_alignment_prep/
+  │   ├─ scripts/             # We supply these in this repo
+  │   ├─ raw_fasta/           # Storing your fasta downloaded from NCBI or Ensembl
+  │   └─ treeval-datasets.tsv # Organism, common_name, clade, family, group, link_to_data, notes
+  │
+  ├─ synteny/
+  │   └─ {classT}
+  │
+  ├─ treeval_yaml/ # Storage folder for you yaml files, it's useful to keep them
+  │
+  └─ treeval_stats/ # Storage for you treeval output stats file whether for upload to our repo
+
+```
+
+`classT` can be your own system of classification, as long as it is consistent. At Sanger we use the below, we advise you do too. Again, this value, that is entered into the yaml (the file we will use to tell TreeVal where everything is), is used to find gene_alignment_data as well as syntenic genomes.
+
+![ClassT](../docs/images/SangerClassT.png)
+
+##### Synteny
+
+For synteny, below the `classT` variable you should store the full genomic fasta file of any high quality genome you want to be compared against.
+
+For bird we recommend the Golden Emu and the Finch, which can be downloaded from NCBI. Rename, these files to something more human readable, and drop them into the `synteny/bird/` folder. Any TreeVal run you now perform where the `classT` is bird will run a syntenic alignment against all genomes in that folder. It would be best to keep this to around three. Again, this is something we could expand on with the `common_name` field if people want in the future, submit a feature request.
+
+##### Gene_alignment_data
+
```

Comment:
Almost the entire section here seems to be a repeat of `genealignmentsynteny.md`. Can't you just link to it like for PacBio after ?

Reply:
You're right. Removed.